master/reference/public-cloud/aws.md

```diff
@@ -0,0 +1,113 @@
+one another using `calicoctl node status`.
+
+#### Allow Traffic Within a Single VPC Subnet
+
+By default, each EC2 instance performs source/destination checks on inbound
+and outbound traffic. This ensures that the instance is the source or
+destination of any traffic it sends or receives.
+
+Since Calico allocates a unique IP address to each container that isn't the IP
+of its host, and since Calico doesn't encapsulate container traffic by default,
+traffic to/from containers will be dropped by the AWS src/dst checks
+
+This reverse-path filtering is enabled by default, but can be disabled on each
+host, allowing Calico traffic to pass through unaffected **between hosts within
+the same VPC subnet.**
+Ensure you follow
+[AWS documentation on disabling source/destination checks](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_NAT_Instance.html#EIP_Disable_SrcDestCheck).
+
+#### Allow Traffic Across Different VPC Subnets
+
+Container traffic that traverses different VPC subnets will pass through
+a VPC routing table. If this VPC routing table does not know how to route
+container/pod IPs, it will drop the traffic. There are two ways two prevent this from
+happening:
+
+1. Encapsulate Container Traffic
+
+   Container traffic can be enapsulated with its host IP so that the VPC
+   never sees the container IPs, allowing standard ec2 routing to take over.
+
+   Turn on traffic masquerading in pool settings by enabling:
+
+   - `ipip` for container-to-container traffic.
+   - `nat-outgoing` for container-to-ec2-instance traffic.
+
+   See [pool configuration]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool) for information on how to enable this for IP pools.
+
+2. Configure the VPC Routing Table
+
+   Alternatively, the VPC routing table can be programmed with container/pod IP
+   routes so that it can route container traffic natively.
+   This can be accomplished in two ways:
+
+   1. Deploy [canal][canal] and select the [aws-vpc backend][aws-vpc-backend],
+      which interacts with AWS api's to automatically program the routing table.
+
+   2. Manually program the routing table (not recommended).
+
+   >Note: [AWS VPC Routing tables are limited to 50 programmable routes](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Appendix_Limits.html#vpc-limits-route-tables).
+
+#### Allow Traffic Across Peered VPCs
+
+AWS drops container traffic when it passes through
+[peered VPCs](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/vpc-peering.html).
+This behavior can not be turned off.
+
+To circumvent this issue, enable `ipip` and `nat-outgoing` on your IP pools ([as discussed above](#allow-traffic-across-different-subnets))
+
+#### Allow Workload-to-WAN Traffic
+
+In a private cloud where operators maintain full control of the networking
+fabric, NAT typically takes place at the edge router.
+
+AWS does not support NAT on outbound traffic from unknown sources
+(i.e. container/pod IPs).
```

Comment:
super picky nit, but this should be `e.g` instead of `i.e`

Might be easier to word as "AWS performs outbound NAT on traffic with the source address of a known EC2 instance, but does not support NAT on traffic from container/pod IP addresses.